core/src/main/scala/quasar/plugin/jdbc/datasource/JdbcDatasource.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Precog Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.plugin.jdbc.datasource
+
+import quasar.plugin.jdbc._
+import quasar.plugin.jdbc.implicits._
+
+import scala.{Boolean, None, Option, Some}
+import scala.util.{Left, Right}
+
+import java.lang.Throwable
+
+import cats.Defer
+import cats.data.{Ior, NonEmptyList}
+import cats.effect.{Bracket, Resource}
+import cats.implicits._
+
+import doobie._
+import doobie.implicits._
+
+import fs2.{Pull, Stream}
+
+import quasar.api.resource.{ResourceName, ResourcePath, ResourcePathType => RPT}
+import quasar.api.datasource.DatasourceType
+import quasar.connector.QueryResult
+import quasar.connector.datasource.{LightweightDatasourceModule, Loader}
+import quasar.qscript.InterpretedRead
+
+import shims.equalToCats
+
+final class JdbcDatasource[F[_]: Bracket[?[_], Throwable]: Defer] private (
+    xa: Transactor[F],
+    discovery: JdbcDiscovery,
+    val kind: DatasourceType,
+    val loaders: NonEmptyList[Loader[Resource[F, ?], InterpretedRead[ResourcePath], QueryResult[F]]])
+    extends LightweightDatasourceModule.DS[F] {
+
+  def pathIsResource(path: ResourcePath): Resource[F, Boolean] =
+    resourcePathRef(path).fold(false.pure[Resource[F, ?]]) {
```

Comment:
More specifically, here, if the `path` contains a schema then we'll discover the table with the given name in that schema.